app/views/report_templates/welcome.html.erb

```diff
@@ -1,12 +1,13 @@
+
 <% content_for(:title, _("Report Templates")) %>
-<div class="blank-slate-pf">
-  <div class="blank-slate-pf-icon">
-    <%= icon_text("file-text", "", :kind => "fa") %>
-  </div>
-  <h1><%= _('Report Templates') %></h1>
-  <p><%= _("The Report Templates provides a way to create textual reports out of Foreman data. When the template is rendered, it loads data from database and renders the result. Based on the template, the output format can be csv, json, yaml or any custom textual format") %></p>
-  <p><%= link_to _('Learn more about this in the documentation.'), documentation_url("4.11ReportTemplates")%></p>
-  <div class="blank-slate-pf-main-action">
-    <%= new_link(_("Create Report Template"), :class => 'btn-lg') %>
-  </div>
-</div>
+
+<% description = _("The Report Templates provides a way to create textual reports out of Foreman data. When the template is rendered, it loads data from database and renders the result. Based on the template, the output format can be csv, json, yaml or any custom textual format")%>
+<% action = { :title => _("Create Report Template"), :url => new_report_template_path } if authorized_for(:action => :new)%>
```

Comment:
Ah, that makes sense actually - `.find` (and all other [`Capybara::Node::Finders`](https://www.rubydoc.info/github/jnicklas/capybara/Capybara/Node/Finders) continuously retries to find the element until `default_max_wait_time` is reached, while [`Capybara::Node::Matchers`](https://www.rubydoc.info/github/jnicklas/capybara/Capybara/Node/Matchers) like `has_link?` only try once and fail if the element isn't there (such as when the check is done before all of the react code finished rendering). The downside of using finders and not matchers is that they can be much slower, especially when they fail - since they will always wait [20 seconds](https://github.com/theforeman/foreman/blob/develop/test/integration_test_helper.rb#L32) before failing.
In the case of the first check on a react page it makes sense to use a finder, though in this case we could use 
```
assert page.find_link('Create Report Template')
```
to maintain the same behaviour (i.e. failing if the link isn't there).
This comment should be have been made on the test though 😉

Reply:
I also thought it's about the time the react component takes to load into the DOM, though while testing it locally and printing the `page.body` content before the assertion, the <a> tag was there,
one thing I noticed in the dif between this PR and before, is that the <a> tag doesn't have an href,
and there's a chance capybara is looking for that, as most links have an `href`.
but let's see if that works :)

Reply:
good find! looks like patternfly `Button` component indeed creates an `a` tag with no `href` but rather with an event handler to handle the click.

Reply:
That sounds like a bad design in Patternfly. Is there a (good) reason they do that?

Reply:
I assume it's because you don't always want the browser to handle the navigation, for example if you use the react router to do front-side routing.

Reply:
Or actually it can take an `href`, it's just our `DefaultEmptyState` that implements it with a click handler - https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/EmptyState/DefaultEmptyState.js#L28